### ISSUE TITLE
ETH transaction details are shown in wallet screen when registering ENS username

### ISSUE THREAD

**Issue by asemiankevich** [status-im/ens-usernames#11]
When user initiates the transaction to register ENS name for 50 SNT charge this is expected wallet screen:

![image](https://user-images.githubusercontent.com/28752642/45479297-c7aaed00-b74e-11e8-93e1-2c35e6f4038b.png)

As an actual result i see ETH transaction which confuses me

![image](https://user-images.githubusercontent.com/28752642/45479488-50c22400-b74f-11e8-8c9b-499c3dae6152.png)

**Comment by bgits** [comment]
Would implementing this EIP help?

https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md

**Comment by dmitryn** [comment]
>Would implementing this EIP help?

> https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md

@bgits yes, it should help with UX. Currently status wallet receives tx data:
```json
{"jsonrpc":"2.0","id":18,"method":"eth_sendTransaction","params":[{"data":"0x095ea7b30000000000000000000000009e183bc54bb4f3cca1a478ca6f2c3edc37b60478ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff","from":"0x511dd326814ea99881c08e4ba43a1463624e9aa7","gasPrice":"0x3b9aca00","to":"0xc55cf4b03948d7ebc8b9e8bad92643703811d162"}]}
```
to enable SNT allowance and the issue is user don't see what he's actually signing. The same issue happens when we ask to transfer 50 SNT.
